src/components/github-htmlview.component.js

```diff
@@ -233,6 +235,11 @@ export class GithubHtmlView extends Component {
             </Text>
           </View>
         ),
+        del: (node, index, siblings, parent, defaultRenderer) => (
+          <Text key={index} style={styles.del}>
+            {defaultRenderer(node.children, node)}
+          </Text>
+        ),
```

Comment:
This is not needed since it's the default behavior of `<HTMLView />` (it will read the `del` style from its `stylesheet` prop).